app/controllers/settings/exports_controller.rb

```diff
@@ -15,32 +12,18 @@ def show
   end
 
   def download_following_list
-    @accounts = current_account.following
+    export_data = Export.new(current_account.following).to_csv
```

Comment:
If you are replacing these with own controllers, why are these methods still in here?